### PR TITLE
chore: freeze 1.1

### PR DESCRIPTION
**Pull request by esavier**

### CODE REVIEW

**Review by luke-biel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# This project is suspended
```
Comment (smores56) [inline README.md:1]: Title case?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+The decision to suspend the project was taken on 2021-09, and therefore we halted all work related to it.
+This commit will produce version 1.1, however this is still open source, so it can be resurrected later or forked at any time.
```
Comment (smores56) [inline README.md:4]: Saying "this commit" feels weird to me. It seems like saying "we left it at 1.1" or something to that effect would be better. Less of an implementation detail, so to say.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+The team will try to track the community-generated issues that will be added to the official and resolve those if possible.
```
Comment (smores56) [inline README.md:5]: The official what?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
+- Error reporting for materialization (#663)
+- pass (schema_id, object_id) pair to materializer general (#660)
+
+### Documentation:
+- performance testing process (#719)
+- GraphQL (#725)
+- Stability docs (#706)
+- short description of resolve tree (#713)
+
+### RFCs
```
Comment (smores56) [inline CHANGELOG.md:35]:
```suggestion
### RFC's
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,57 @@
 # Common Data Layer Changelog
 
+## 1.1.0
+
+### Breaking changes:
```
Comment (smores56) [inline CHANGELOG.md:5]:
```suggestion
### Breaking Changes:
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,57 @@
...
+None! :D
+Also we provided document of which api is stable and which is not.
+
+### Api changes:
```
Comment (smores56) [inline CHANGELOG.md:9]:
```suggestion
### API Changes:
```

**Review by smores56 — CHANGES_REQUESTED** [review]

**Pull request merged**